src/index.tsx

```diff
@@ -12,42 +12,14 @@ const App = (): JSX.Element => {
   const stateManager = useReducer(StateUpdaters.reducer, Config.defaultState);
   const [state, dispatch] = stateManager;
 
-  // Promise to get markers for served build
-  useEffect(() => {
-    const fetchMarkers = async () => {
-      const response = await fetch("/static/waypoints.json");
-      const markers = await response.json();
-      dispatch({ type: "setRows", value: markers });
-    };
-    fetchMarkers();
-  }, [dispatch]);
-
-  // Only rerenders when current combinedRows differs from previous.
-  useEffect(() => {
-    dispatch({
-      type: "updateCombinedRows",
-      value: StateUpdaters.getUpdatedCombinedRowsByZoom(
-        state.rows,
-        state.combinedRows,
-        state.mousePosition.zoom
-      ),
-    });
-  }, [
-    dispatch,
+  // Update markers from spreadsheet and render via zoom level
+  useEffect(StateUpdaters.getJsonMarkers(dispatch), []);
```

Comment:
I wanted to factor out useEffect lambdas into named functions since they were getting more complicated, and they would benefit from an explanatory name